frontend/public/components/utils/kebab.tsx

```diff
@@ -506,6 +511,7 @@ export type KebabOption = {
   // Eg. `Menu 1/Menu 2/Menu 3`
   path?: string;
   icon?: React.ReactNode;
+  isExternalLink?: boolean;
```

Comment:
@rawagner @pcbailey hi, 
does changing line [L499](https://github.com/openshift/console/pull/6078/files#diff-44eaa3d336fe8f3b11ac4f13e1ebf60dL499) to:
`label: string | React.ReactElement;`

and then using label to pass a component with the icon included makes sense to you ?

Reply:
`label: React.ReactNode` would be the best

Reply:
I tried making this change, but hit an issue. The `label` field is used to create the `key` for the items in `KebabMenuItems`. I tried to cast the label to a string, but it caused a type conflict for the `key` in `frontend/packages/dev-console/src/components/topology/components/nodeContextMenu.tsx` on line 34. I could cast that one to a string as well, but that will cause the PR to spread across three different packages. 

@yaacov  @rawagner Thoughts?

Reply:
Nothing better than just using `index` comes to my mind

```
{_.map(options, (o, index) => (
      <li key={index}>
```

Reply:
Gotcha. I've seen where it's considered a discouraged practice and wasn't sure if I should go that route or not. Thanks! =)

Reply:
It's still going to require approval for three different packages, though.